docs/exporting-configuration/terraform-provider-compatability.md

```diff
@@ -0,0 +1,16 @@
+# Ping CLI Platform Export Terraform Provider Support
+
+The platform export command exports configuration-as-code packages for the Ping Platform. The CLI can export
+Terraform HCL to use with released Terraform providers.
+
+The following describes the CLI's support of each of these Ping Terraform providers.
+
+| Service             | Support | Versions |
+| ------------------- | ---- | ------- |
+| PingOne | :white_check_mark: | >= 1.1.1 |
+| PingFederate | :white_check_mark: | >= 1.0.0 |
```

Comment:
Need to add both "PingDirectory" and "DaVinci" as :x: (TBD)
Both PingOne and PingFederate should be partial support (in progress) as they're not yet complete

Also work a link somewhere to the hcl-export-compatability.md file

Reply:
Added!